kubernetes/k8_create.yml.erb

```diff
@@ -0,0 +1,60 @@
+# Deployment for create and redis service
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: create-deployment
+  labels:
+    app: create
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: create
+  template:
+      metadata:
+        labels:
+          app: create
+      spec:
+        containers:
+          - name: create-worker
+            image: "docker.ocf.berkeley.edu/create:<%= version %>"
+            imagePullPolicy: Always
+            ports:
+              - containerPort: 6378
```

Comment:
Why is this needed? Is this the port Celery listens on? If so, don't we need a k8s `service` as well?

Reply:
Not strictly necessary, mostly just informational.

Reply:
What is it though?